docs/user-guide/file-management/file-management-faq/README.md

```diff
@@ -8,7 +8,10 @@ keywords: files, attachment, Nextcloud, OneDrive, SharePoint, FAQ
 
 # File Management FAQs
 
-## Why am I not allowed to see/read a certain file in OneDrive/SharePoint?
+## Why am I not allowed to see/read a certain file in OneDrive/SharePoint or Nextcloud?
 
 It is possible that you lack the necessary permissions to view a certain file. In this case please contact your administrator. 
-Another explanation may be that you have been removed from a project in OpenProject, which will also mean that you lost your viewing or reading privileges in OneDrive/SharePoint project folders.
\ No newline at end of file
+Another explanation may be that you have been removed from a project in OpenProject, which will also mean that you lost your viewing or reading privileges in OneDrive/SharePoint or Nextcloud project folders.
```

Comment:
❓ There is another reason: A project admin can revoke your permission to view files on file storages within OpenProject.

Should we add this?

Reply:
yes, good point! I'll add it.